src/containers/SubjectPage/components/SubjectEditorChoices.tsx

```diff
@@ -25,23 +26,31 @@ const getResourceTypeName = (resource, t) => {
   ) {
     return t('subjectPage.editorsChoices.unknown');
   }
-  return resource.resourceTypes[0].name;
+  return resource.resourceTypes[0]!.name;
 };
 
+interface Props {
+  editorsChoices: Array<GQLTaxonomyEntity | null> | undefined;
+  narrowScreen?: boolean;
+  wideScreen?: boolean;
+  locale: LocaleType;
+}
+
 const SubjectEditorChoices = ({
   editorsChoices,
   narrowScreen,
   wideScreen,
-  locale,
   t,
-}) => {
+}: Props & tType) => {
   if (!editorsChoices) {
     return null;
   }
 
   const editorsChoicesResources = editorsChoices
     .filter(hasContentUri)
-    .map(resource => ({
+    .filter(x => x !== null)
+    //@ts-ignore
+    .map((resource: GQLTaxonomyEntity) => ({
```

Comment:
Nå som du har fjernet `null` fra GQL typer, kan du også fjerne `| null` fra editorsChoices slik at du kan fjerne `ts-ignore` herfra. Samtidig kan du også flytte på rekkefølgen på disse to filter funksjonene, slik at vi fjerner eventuelle `nuller` før vi sjekker `hasContentUri`